server.js

```diff
@@ -28,7 +29,7 @@ app.use(bodyParser.json());
 
 app.use('/public', express.static(path.join(__dirname, 'public')));
 
-app.get('/suncalc.js', (req, res) => {
+app.get('/suncalc.js', (_, res) => {
```

Comment:
What's going on here and the other, similar `app.get`s?

Reply:
This is just standard notation for an unused variable in a callback. It gets better highlighting in the editor

Reply:
Ah ok, that makes sense now